buildkite/scripts/generate-diff.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Base against origin/develop by default, but use pull-request base otherwise
```

Comment:
can you update the comment to point to master by default too?

Reply:
Shouldn't this default to `compatible` instead of `master`?